installTerraInVm.sh

```diff
@@ -38,9 +38,11 @@ https://apt.releases.hashicorp.com $(lsb_release -cs) main" | \
 fi
 
 # Check if apt needs updating after potential repository addition
-if test -f /etc/apt/sources.list.d/hashicorp.list && find /etc/apt/sources.list.d/hashicorp.list -mmin -5; then
-    echo 'Running apt update due to recent repository changes...'
-    sudo apt update
+if (apt-cache show terraform)
+  then 
+  echo 'terrform package is already up to date'
```

Comment:
the message should be something like "terraform is in the apt cache and available to be installed"